src/ethereum/ethPlugin.js

```diff
@@ -164,6 +171,7 @@ export class EthereumPlugin extends CurrencyPlugin {
           contractAddress,
           currencyName,
           multiplier,
+          denominations: [{ name: currencyCode, multiplier }],
```

Comment:
Does the GUI need denominations? I don't think it makes sense to return this, now that I'm thinking about it more.